src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -113,7 +113,9 @@
         // scalability (VP9). Exotic cases might do simulcast + spatial
         // scalability.
 
+        int resolution = 180;
```

Comment:
This `resolution` should, in theory, be included in the SDP answer of a sender, while the thumbnail resolution constraint in the bitrate controller should come from the receiver as a data channel message (as it is requested here https://github.com/jitsi/jitsi-videobridge/issues/147) or even in the answer of a receiver.

So these two refer to different things and I feel uncomfortable using the same constant. How about I use another constant here (and I also include the above explanation)?